translations/ui-checkout/en.json

```diff
@@ -90,12 +90,12 @@
   "messages.userHasNoBarcode": "User does not have a barcode.<br></br>A user barcode is required to check out an item.",
   "messages.itemIsNotLoanable": "<strong>{title} ({materialType})</strong> (Barcode: {barcode}) is not <strong>loanable</strong> according to the <strong>{loanPolicy}</strong> loan policy",
   "messages.itemHasOpenLoan": "Cannot check out item that already has an open loan.",
-  "messages.itemLimitPatronGroupMaterialTypeLoanType": "Patron has reached maximum limit of 1 items for combination of patron group, material type and loan type.",
-  "messages.itemLimitPatronGroupMaterialType": "Patron has reached maximum limit of 1 items for combination of patron group and material type.",
-  "messages.itemLimitPatronGroupLoanType": "Patron has reached maximum limit of 1 items for combination of patron group and loan type.",
-  "messages.itemLimitMaterialTypeLoanType": "Patron has reached maximum limit of 1 items for combination of material type and loan type.",
-  "messages.itemLimitMaterialType": "Patron has reached maximum limit of 1 items for material type.",
-  "messages.itemLimitLoanType": "Patron has reached maximum limit of 1 items for loan type.",
+  "messages.itemLimitPatronGroupMaterialTypeLoanType": "Patron has reached maximum limit of {itemLimit} items for combination of patron group, material type and loan type.",
+  "messages.itemLimitPatronGroupMaterialType": "Patron has reached maximum limit of {itemLimit} items for combination of patron group and material type.",
+  "messages.itemLimitPatronGroupLoanType": "Patron has reached maximum limit of {itemLimit} items for combination of patron group and loan type.",
+  "messages.itemLimitMaterialTypeLoanType": "Patron has reached maximum limit of {itemLimit} items for combination of material type and loan type.",
+  "messages.itemLimitMaterialType": "Patron has reached maximum limit of {itemLimit} items for material type.",
+  "messages.itemLimitLoanType": "Patron has reached maximum limit of {itemLimit} items for loan type.",
```

Comment:
`limit of {itemLimit} items for combination...` won't be correct for single values i.e. "1 item". Instead, use the [`plural` format](https://formatjs.io/docs/react-intl/components/#formattednumber) like this:
```
limit of {itemLimit, plural, one {1 item} other {{itemLimit} items}} for combination...
```